test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -49,14 +64,9 @@ jest.mock('@folio/stripes/components', () => ({
   Headline: jest.fn(({ children, ...rest }) => <div {...rest}>{children}</div>),
   Icon: jest.fn((props) => (props && props.children ? props.children : <span />)),
   IconButton: jest.fn(({
-    buttonProps,
-    // eslint-disable-next-line no-unused-vars
-    iconClassName,
     ...rest
```

Comment:
in this case we do not need `...rest` we can just write like this `jest.fn((props) => {})`

Reply:
Good catch. Fixed